### DISCUSSION TITLE
Behavioural parameters: upper bound

### DISCUSSION THREAD

**Discussion by sofiagilardini** [AdaptiveMotorControlLab/CEBRA#124] [Q&A] [answered]
Hello, I was wondering if there was a specified upper bound for the number of behavioural inputs (continuous) for training a CEBRA model? 

Thank you!

**Comment by MMathisLab** [comment, marked as answer]
Hi there! You should not have more labels than input (neurons). The goal of cebra is to effectively do nonlinear ICA, so to separate labels into independent components ; in practice you should consider the input data, as it might be very simple or complex and need more or less inputs- if that makes sense!
